### ISSUE TITLE
[Feature Request] Auto upload from folder 

### ISSUE THREAD

**Issue by Schrolli91** [astubenbord/paperless-mobile#279]
### Description

It would be nice if it's possible to auto upload files from a local folder, like the paperless comsume folder.

I can scan from my Canon printer to a local folder on my smartphone. At the moment I have to share the file to paperless with the "paperless share" app.

The idea is, that the new scanned files will be auto uploaded to paperless from that folder. After that it can be deleted or maybe moved to another folder.

This will improve and speed up the process because I have not to share every file or upload manually by paperless mobile or over the web ui.

**Comment by astubenbord** [comment]
Any reason why you cannot upload these documents directly to the paperless consumption folder? Maybe you can set up an SMB share.

**Comment by Schrolli91** [comment]
Good point, I will give it a try tomorrow, if that will work as I want it.

But I'm not sure if that's possible on android?

**Comment by astubenbord** [comment]
I assume both your printer and server are in the same (home) network?

**Comment by Schrolli91** [comment]
printer and smartphone are in the local network.
paperless itself runs on an VM behind a proxy in the www

**Comment by Schrolli91** [comment]
I have only found a solution via an separate app that will upload the files via webdav.
But a good native solution in android seems not possible to me.

**Comment by astubenbord** [comment]
Okay, I see. The app does in fact scan a certain folder on startup and loads all files in there. Maybe I can add a setting so users can select the folder from which files are consumed? This might also be along the lines of what is suggested in #198.

**Comment by Schrolli91** [comment]
Sounds good to me.
